### ISSUE TITLE
Question: What is this project's goals and future vision?

### ISSUE THREAD

**Issue by Aukstkalnis** [tinygo-org/tinygo#194]
Hi,

This project looks interesting. It seams only @aykevl and @deadprogram are maintaining tinygo. So I am curious, what is your motivation and future vision for developing embedded GO? It is hedge mountains of work. MacOS, Linux, Windows X, Windows Y, Windows Z, ARM, LLVM and numerous number of MCU's... All these platforms require lots of time for learning, implementing, testing and debug. Are you doing this in free time? For fun? For now there is not much users of tinygo, but if you keep this up, then more things will go on, more bug reports, more time debugging, fixing, testing - less time developing. Then things would start looking boring, tiresome and motivation can fade away.
So, to sum it up, are you trying to create project that could take over embedded world, or are you doing it for fun?

Thanks.

**Comment by deadprogram** [comment]
Global takeover, of course.

Thanks for asking. Any other questions?

**Comment by Aukstkalnis** [comment]
Ok... I guess my question was very boring and funny😅 So as I understand you want to create new language for embedded systems to replace C. OK, that's cool. Maybe I will give it a tray :)

**Comment by deadprogram** [comment]
Boring, no. Funny, yes. :)

Hope you do give it a try!
